docs/reference/mapping.asciidoc

```diff
@@ -68,6 +68,14 @@ via the <<multi-fields>> parameter.
 [float]
 === Settings to prevent mappings explosion
 
+Defining too many fields in an index or having too many nested documents or too many fields that contain nested
+documents are conditions that can lead to a mapping explosion, a case that occur when a cluster run ot of memory
+and finds itself in a difficult situation to recover. This problem may be more common than expected.
+As an example, consider a situation in which every new document inserted introduced new fields.
+This is entirely legit possibility considering that Elasticsearch has the ability to be schema-less, which means that
```

Comment:
This is possible considering with dynamic mappings. Every time a document contains new fields, those will end up in the index's mappings.